ProjFS.Mac/PrjFS.xcodeproj/xcshareddata/xcschemes/Build All.xcscheme

```diff
@@ -26,7 +26,25 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      codeCoverageEnabled = "YES"
+      onlyGenerateCoverageForSpecifiedTargets = "YES"
       shouldUseLaunchSchemeArgsEnv = "YES">
+      <CodeCoverageTargets>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "4A8C139F21F23EE800002878"
+            BuildableName = "libPrjFSKextTestable.a"
+            BlueprintName = "PrjFSKextTestable"
+            ReferencedContainer = "container:PrjFS.xcodeproj">
+         </BuildableReference>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "F5E39C7621F1118D006D65C2"
+            BuildableName = "PrjFSKextTests.xctest"
+            BlueprintName = "PrjFSKextTests"
+            ReferencedContainer = "container:PrjFS.xcodeproj">
+         </BuildableReference>
+      </CodeCoverageTargets>
```

Comment:
We need to limit code coverage to specific targets, as coverage instrumentation is not supported for kexts and will screw up the build.

Reply:
Where in Xcode is the coverage shown?

I'm on you're branch and I've confirmed that the code coverage is set to these two projects (in the GUI), but I've not been able to figure out where I need to go to see anything about the coverage.

Reply:
@nickgra helped me figure this out.  I had to:

1. View->Navigators->Show Report Navigator
2. Run the tests
3. Select (in the tree-control on the left) Build All -> Test -> Coverage
4. Check "Show Test Bundles" in the top-right corner

Reply:
In case someone else hits this, another setting that might need adjusting is Editor -> Show Code Coverage

Reply:
Apologies, all that seemed to be necessary my end was to actually run the tests, but perhaps that was down to some previous Xcode state.